legal-api/src/legal_api/services/authz.py

```diff
@@ -88,3 +90,100 @@ def has_roles(jwt: JwtManager, roles: List[str]) -> bool:
     if jwt.validate_roles(roles):
         return True
     return False
+
+
+ALLOWABLE_FILINGS_ACTIVE: Final = {
+    'alteration': {
+        'staff': ['BC', 'BEN'],
+        'user': ['BC', 'BEN'],
+    },
+    'annualReport': {
+        'staff': ['CP', 'BEN'],
+        'user': ['CP', 'BEN'],
+    },
+    'changeOfAddress': {
+        'staff': ['CP', 'BEN'],
+        'user': ['CP', 'BEN'],
+    },
+    'changeOfDirectors': {
+        'staff': ['CP', 'BEN'],
+        'user': ['CP', 'BEN'],
+    },
+    'correction': {
+        'staff': ['CP', 'BEN'],
+    },
+    'courtOrder': {
+        'staff': ['CP', 'BC', 'BEN', 'CC', 'ULC', 'LLC'],
+    },
+    'dissolution': {
+        'staff': ['CP', 'BC', 'BEN', 'CC', 'ULC', 'LLC'],
+        'user': ['CP', 'BC', 'BEN', 'CC', 'ULC', 'LLC'],
+    },
+    'incorporationApplication': {
```

Comment:
All entries for different role look the same to me. I feel this should be other way round. Based on the user role and entity state, you can return the filing types allowed. There could be one or more filing types that is condition based. Those alone can be added based on condition is what I feel

Reply:
I agree with Lekshmi. This is essentially a 3-dimensional array of values but it feels like it might not easily extensible currently -- what if we add states (eg, NIGS) or other conditions?